shared/us-east-1/tools-costs-email-notifications/src/script.py

```
@@ -251,39 +376,89 @@ def lambda_handler(event, context):
+        if ce_client is None:
+            logger.warning(f"Skipping account {account_name} - failed to create CE client")
+            if account_name not in failed_accounts:
+                failed_accounts.append(account_name)
+            continue
+
         # Check if the account_name exists in tag_costs
         if account_name in tag_costs:
             prev_cost = tag_costs[account_name]
         else:
             # Handle the case where tag_costs doesn't contain the key
             prev_cost = Decimal(0)
```

Ruff (0.14.10)

Local variable `prev_cost` is assigned to but never used

Remove assignment to unused variable `prev_cost`

(F841)